pfb-analysis/scripts/entrypoint.sh

```diff
@@ -52,6 +52,18 @@ then
     popd
 fi
 
+# If given a URL for the OSM file, dowload and unzip it. Overrides PFB_OSM_FILE.
+if [ "${PFB_SHPFILE_URL}" ]
```

Comment:
Oops, copy/paste/modify error.  Just pushed a fix.

I put zipped copies of the OSM files in the test files bucket, actually, because they're so big (220MB for lower Manhattan) and they deflate by about 90%.